tools/createValidate.js

```diff
@@ -0,0 +1,132 @@
+const fs = require("fs");
+
+const NORMAL_TYPES = ["number", "string"];
+const NUMBERS = ["0", "1"];
+
+// TODO: rewrite this to use regex
```

Comment:
Flagging - Should this happen before merging or maybe create an issue to track it?

Reply:
Should be an issue - I don't think it's worth making this change right now, the script works just fine.